src/controllers/itemdetailpage.js

```diff
@@ -2078,7 +2078,7 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "userSetti
         bindAll(view, ".btnCancelTimer", "click", onCancelTimerClick);
         bindAll(view, ".btnDeleteItem", "click", onDeleteClick);
         bindAll(view, ".btnDownload", "click", onDownloadClick);
-        view.querySelector(".btnMoreCommands i").innerHTML = "&#xE5D3;";
+        view.querySelector(".btnMoreCommands i").innerHTML = "&#xe5d4;";
```

Comment:
we'd better use classes rather than inline stuff, inline stuff can break on WebOS and other obscure things

Reply:
hex will work, but I removed this line :p - moved icon to HTML template

Reply:
okay, ignore me here then :)